js/components/ConsoleToolbar.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { messageClear } from '../actions/AppActions';
+import { connect } from 'react-redux';
+
+class ConsoleToolbar extends Component {
+  render() {
+    return (
+      <button onClick={this.clear.bind(this)}>Clear</button>
+    );
+  }
+
+  clear() {
```

Comment:
We decided last week in the meeting to avoid having methods on components. Instead, this could be a function declared inside of the render() method. That would have the additional advantage of not having to bind.